brd-ios/breadwallet/Assets.xcassets/Redesign/icons/SwitchPlaces.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "SwitchPlaces.pdf",
```

Comment:
asset not required... it exist in the existing resources (and is called swap)

Reply:
I am removing the old asset because it is easier to use the one I extracted (already has the rounded corners). also the old one does not match the designs anymore.

![image](https://user-images.githubusercontent.com/99484553/177487835-1c9e4640-47e6-45c5-b7a6-eb0616c1e140.png)